sql/firefox_kpi_dashboard_v1.sql

```diff
@@ -1,14 +1,13 @@
 CREATE TEMP FUNCTION
-  udf_sum_buckets_with_ci_js(a ARRAY<INT64>)
+  udf_sum_buckets_with_ci_js(n_buckets INT64, counts_per_bucket ARRAY<INT64>)
```

Comment:
By passing in the number of buckets, we avoid accidentally under-counting when there are empty buckets, leading to an array smaller than 20 elements.